### DISCUSSION TITLE
Can you generate nested link pages dynamically?

### DISCUSSION THREAD

**Discussion by mackerel225** [requarks/wiki#7382] [Help / Questions] [answered]
Consider this page layout:

- root
- - Python Guides < -------------------- Can we modify this page to dynamically add child pages as links, i.e. `Python Basics & Advanced`
- - - Python Basics
- - - Python Advanced

The manual approach for now is to do this:
```
[Python Basics](/python/basics)
[Python Advganced](/python/advanced)
...
```

It becomes tiring if you have over 100 pages

**Comment by fnafox** [comment, marked as answer]
I believe this feature (dynamic links) is planned for the v3 release. To my knowledge there's no way to do it currently aside from hacking together an external script that updates the page using the api.
